scripts/benchmark.py

```diff
@@ -1,13 +1,11 @@
 from manticore import Manticore
+from manticore.utils.helpers import pretty_print_results
```

Comment:
what if instead of a helper, we make a `.dump()` or `.to_str()` method on `ProfilingResults` itself? (i'm not sure of the most idiomatic name)